app/assets/styles/partials/components/_relationship.scss

```diff
@@ -55,3 +55,18 @@
 .relationship__field {
   display: block;
 }
+
+
+// TO BE ADDED TO PL AND REMOVED FROM HERE
+.question--repeatinganswer {
+  .question__answer {
+    width: 19.5rem;
+  }
+}
+
+.btn--wa {
```

Comment:
This (`btn--wa`) would be better handled as a utility class as it's a single CSS rule that deals with layout, not anything that is specific to buttons. It could also then be re-used elsewhere and controlled via breakpoint prefixes.

Reply:
So extract from here and turn into a `.u-wa` class in the `./assets/sass/utilities/_utilities.scss` for greater utility across the pattern library?